### ISSUE TITLE
screenshot created but not copied to clipboard with `jpeg` but `png` works

### ISSUE THREAD

**Issue by kohane27** [ObserverOfTime/mpv-scripts#10]
### Script(s)

`clipshot.lua`

### Description

Hello there. Hope you're doing well. Thank you for creating this script; it's very handy.

However, I had trouble using it. After pressing the shortcut, when I tried to paste it into applications (e.g., Obsidian) with `Ctrl-v` there's nothing to paste.

### Possible Fix

I found the cause. The following works:

```lua
NAME = "mpv-screenshot.png"
SHOT = "/tmp/" .. NAME
CMD = { "xclip", "-sel", "c", "-t", "image/png", "-i", SHOT }
```

The following doesn't work:

```lua
NAME = "mpv-screenshot.jpeg"
SHOT = "/tmp/" .. NAME
CMD = { "xclip", "-sel", "c", "-t", "image/jpeg", "-i", SHOT }
```

For both extensions, screenshots are created in `/tmp/`, but only the png version above are copied to clipboard.

### Steps to Reproduce

Same issue with xclip 0.13 and manual compilation from [xclip/master](https://github.com/astrand/xclip).

Should we fall back to using `png` when it's Xorg?

Thanks:)

### Operating System(s)

- [ ] Windows
- [X] Linux
- [ ] macOS

### MPV version

mpv 0.34.1-dirty

### Lua version

5.4.4

**Comment by ObserverOfTime** [comment]
It's copied properly but can't be pasted for whatever reason.
However, it can be selected through a clipboard manager.

The problem with png is its size, and webp has the same issue as jpeg.

Upstream issue: astrand/xclip#110

**Comment by NaiveInvestigator** [comment]
this also occurs for me when using `wl-copy` too and works if i change it to `png`

> However, it can be selected through a clipboard manager.

Sadly for me i can't, i use the one what comes with KDE but the copied image does not show up for me.

**Comment by ObserverOfTime** [comment]
bugaevc/wl-clipboard#162 ?

**Comment by NaiveInvestigator** [comment]
> It's copied properly but can't be pasted for whatever reason.

after checking https://github.com/bugaevc/wl-clipboard/issues/162 , this does seem  to be the case mb.

`wl-paste --list-types` gives me `image/jpeg` after i copy the image from mpv. 

But weirdly it doesn't show on Klipper (KDE's default clipboard manager) still for me. Will just use `png` for now as i have no issues with the file size personally. TYSM for the help!

**Comment by Justtaway** [comment]
Hello! Kinda necroposting, but suddenly got exact same issue. But it, does not copy to clipboard whatsoever (neither jpeg or png).
I have Win11x64 22631.3296, mpv.net 7.1.1.0 (03.02.2024), mpv v0.37.0-152-gbd5d8e41 (11.01.2024). Screenshot is created in temp folder, but does not go to clipboard. 

I tried enabling logs and got this, the most relevant line, as I believe, is:
`[  15.820][e][cplayer] Starting subprocess failed: init
`
Here is full logs:
[output.txt](https://github.com/ObserverOfTime/mpv-scripts/files/14859615/output.txt)

I tried another script at https://github.com/zc62/mpv-scripts/blob/master/screenshot-to-clipboard.js and it does work (and I got it to screenshot in Temp folder), but I don't know how to make it jpeg instead of png.
